### ISSUE TITLE
Incompatibilities with Ansible 2

### ISSUE THREAD

**Issue by joohoi** [PowerDNS/pdns-ansible#5]
The condition definitions like https://github.com/PowerDNS/pdns-ansible/blob/master/tasks/install.packages-debian.yml#L33 break with Ansible 2, not sure about the cause, but they must have changed something about looping, as the error message is 

```
TASK [PowerDNS.pdns : Install Geo module package] ******************************
fatal: [ns1.0s.fi]: FAILED! => {"failed": true, "msg": "ERROR! The conditional check 'item.startswith('geo')
 and not item.startswith('geoip')' failed. The error was: ERROR! error while evaluating conditional
 (item.startswith('geo') and not item.startswith('geoip')): ERROR! 'list object' has no attribute 
'startswith'\n\nThe error appears to have been in 
'/etc/ansible/roles/PowerDNS.pdns/tasks/install.packages-debian.yml': line 31, column 3, but may\nbe 
elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- 
name: Install Geo module package\n  ^ here\n"}
```

Running the role with Ansible 2.0.0.2 produces the above error message, works perfectly when downgraded to 1.9.4.

**Comment by pieterlexis** [comment]
thanks for reporting this, we'll have a look at this next week.

**Comment by mmitucha** [comment]
Hi, 
I also ran into this issue, so sent a pull request ( #6 ) which works for me.

**Comment by pieterlexis** [comment]
fixed in #6. Thanks @mmitucha

**Comment by mmitucha** [comment]
Wonderful, I'm glad I could help.
